src/main/java/htsjdk/samtools/util/AbstractProgressLogger.java

```diff
@@ -125,22 +149,25 @@ public boolean record(final SAMRecord... recs) {
     public long getElapsedSeconds() { return (System.currentTimeMillis() - this.startTime) / 1000; }
 
     /** Resets the start time to now and the number of records to zero. */
-    public void reset() {
+    public synchronized void reset() {
         startTime = System.currentTimeMillis();
         processed = 0;
         // Set to -1 until the first record is added
         lastStartTime = -1;
         lastChrom = null;
         lastPos = 0;
+        lastReadName = null;
+        countNonIncreasing = 0;
     }
 
     /** Left pads a string until it is at least the given length. */
-    private String pad (String in, final int length) {
-        while (in.length() < length) {
-            in = " " + in;
+    protected static String pad(final String in, final int length) {
```

Comment:
make this default package instead of protected.

Reply:
done